sites/partners/lib/hooks.ts

```diff
@@ -40,10 +46,12 @@ export function useSingleListingData(listingId: string) {
   }
 }
 
-export function useListingsData({ page, limit, userId }: UseListingsDataProps) {
+export function useListingsData({ page, limit, userId, sort }: UseListingsDataProps) {
   const params = {
     page,
     limit,
+    orderBy: sort.filter((item) => item.orderBy).map((item) => item.orderBy),
+    order: sort.filter((item) => item.order).map((item) => item.order),
```

Comment:
@dominikx96 , what's getting filtered here? is it for if orderBy or order are undefined or null?

Reply:
@seanmalbert yes, the goal is to remove undefined values and create an array from these values